### PR TITLE
Simplify start-chunk handling

### PR DESCRIPTION
**Pull request by janpolowinski**
Use the first chunk (alphabetically) as a start-chunk

### CODE REVIEW

File: src/main/java/de/tu_dortmund/ub/data/dswarm/Init.java [hunk 1]
```diff
@@ -76,11 +76,12 @@ of this software and associated documentation files (the "Software"), to deal
 
 	private final Properties config;
 	private final Logger     logger;
-
+	private final String initResourceFileName;
 	private final int cnt;
 
-	public Init(final Properties config, final Logger logger, final int cnt) {
+	public Init(final String initResourceFileName, final Properties config, final Logger logger, final int cnt) {
```
Comment (boferri) [inline src/main/java/de/tu_dortmund/ub/data/dswarm/Init.java]: it's better to use the absolute file path here

File: src/main/java/de/tu_dortmund/ub/data/dswarm/TaskProcessingUnit.java [hunk 2]
```diff
@@ -134,8 +136,11 @@ public static void main(final String[] args) throws Exception {
 
 		// init
 		if(doInit) {
+			
+			// use the first file in the folder for init
+			String resourceFileName = files[0];
```
Comment (boferri) [inline src/main/java/de/tu_dortmund/ub/data/dswarm/TaskProcessingUnit.java]: make use of final ,)

File: src/main/java/de/tu_dortmund/ub/data/dswarm/TaskProcessingUnit.java [hunk 3]
```diff
@@ -225,11 +234,11 @@ public static void main(final String[] args) throws Exception {
 		logger.info(tasksExecutedMessage);
 	}
 
-	private static String executeInit(final String serviceName, final Integer engineThreads) throws Exception {
+	private static String executeInit(String initResourceFileName, final String serviceName, final Integer engineThreads) throws Exception {
```
Comment (boferri) [inline src/main/java/de/tu_dortmund/ub/data/dswarm/TaskProcessingUnit.java]: make use of final ;)

**Comment by janpolowinski** [comment]
Agreed. Done!

**Pull request merged**